XDTransformExample/authoring/build/XMLTransformExample.targets

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+ 
+  <Target Name="CollectConfigFilesTransforms" BeforeTargets="CollectWebConfigsToTransform">
```

Comment:
Add a comment telling the author to rename this, if multiple packages have the same target only 1 will win

@nkolev92 added an `<Error>` to help solve this

Reply:
Pointed to the sample offline. 
https://github.com/NuGet/Samples/blob/master/NuGet.Samples.Interop/NuGet.Samples.Interop.targets#L17

Putting a random placeholder which would make the target not interpretable also works.